templates/my.cnf.j2

```diff
@@ -2,10 +2,18 @@
 #password = your_password
 port = {{ mysql_port }}
 socket = {{ mysql_socket }}
+{% if ansible_os_family == 'Debian' %}
```

Comment:
I'd rather not flag vars based on OS, but rather on MySQL version... is `default-character-set` available in older MySQL versions? If so, we could just leave it without an if...